test.py

```diff
@@ -3,16 +3,9 @@
     Author: Genevieve Hayes <ghayes17@gmail.com>
     License: 3-clause BSD license.
 """
-
 import unittest
-from fitness import OneMax, FlipFlop, head, tail, max_run, Queens, np
-
-# from algorithms import *
-# from discrete import *
-# from fitness import *
-# from decay import *
-# from neural import *
-# import numpy as np
+from fitness import (OneMax, FlipFlop, head, tail, max_run, Queens, np,
```

Comment:
`np` is numpy, not a function\class created in `fitness`. As a result, we should import that with a separate import statement.
```suggestion
import numpy as np
from fitness import (OneMax, FlipFlop, head, tail, max_run, Queens, 
```

Reply:
Good catch, I see that now.